aws/load_balancer/outputs.tf

```diff
@@ -27,3 +27,8 @@ output "lb_target_group_2_name" {
   description = "Load balancer target group 2 name, used by CodeDeploy to alternate blue/green deployments"
   value       = aws_lb_target_group.form_viewer_2.name
 }
+
+output "temporary_token_generated_outside_canada_metric_name" {
+  description = "Metric tracking request made outside of Canada in order to generate a temporary token"
+  value       = one([for x in aws_wafv2_web_acl.forms_acl.rule : one(x.visibility_config.*.metric_name) if x.name == "TemporaryTokenGeneratedOutsideCanada"])
```

Comment:
This is interesting - I'm assuming this is pulling the [metric name from the WAF ACL rule](https://github.com/cds-snc/forms-terraform/pull/174/files#diff-005d484ce6f84f8ddf355649fe1d08cb7af80cc4aaf741dd74bf3f5ef5894ca3R205) right?

Reply:
Absolutely :) It was fun trying to figure out how to fetch that information!